core/src/main/java/feast/core/grpc/CoreServiceImpl.java

```diff
@@ -191,6 +243,40 @@ public void listStores(
     }
   }
 
+  @Override
+  public void applyEntity(
```

Comment:
I think we should start to layer on comments here.

Reply:
Added a small comment, I guess SpecService function has docs on it, not sure if we want to add too much here.